productcat/src/main/java/com/visenze/productcat/android/ImageSearchParams.java

```diff
@@ -322,6 +323,10 @@ private void addDeviceInfoParams(Map<String, List<String>> map,
             this.putStringInMap(map, UC, deviceInfo.getUc());
         }
 
+        if(deviceInfo.getDoNotTrack() != null) {
```

Comment:
this should be a boolean, and we generate "yes" or "no" based on what they select